### PR TITLE
splunk_app_action_1b8c878e3d48f421d2a2932591cc779d

### PR DESCRIPTION
**Pull request by VatsalJagani**

### CODE REVIEW

File: TA-lansweeper-add-on-for-splunk/bin/TA_lansweeper_rh_sites.py [hunk 1]
```diff
@@ -56,46 +57,63 @@ def handleList(self, conf_info):
...
+                        status_code, response.text)
                     if is_expired_response:
                         lansweeper.access_token = is_expired_response['access_token']
                         lansweeper.refresh_token = is_expired_response['refresh_token']
                         # Updating the access token and refresh token in the conf files
                         try:
-                            update_access_token(access_token=is_expired_response['access_token'], refresh_token=is_expired_response['refresh_token'], client_secret=client_secret, session_key=session_key, stanza_name=account_name)
-                            logger.info('Successfully updated the new access token and refresh token in the conf file')
+                            update_access_token(access_token=is_expired_response['access_token'], refresh_token=is_expired_response[
+                                                'refresh_token'], client_secret=client_secret, session_key=session_key, stanza_name=account_name)
```
Comment (mahirchavda) [inline TA-lansweeper-add-on-for-splunk/bin/TA_lansweeper_rh_sites.py]: The formatting looks weird.

Comment (VatsalJagani) [reply TA-lansweeper-add-on-for-splunk/bin/TA_lansweeper_rh_sites.py]: Fixed the formatting issue.

**Review by mahirchavda — APPROVED** [review]

**Comment by VatsalJagani** [comment]
@mahirchavda - Shall we keep this PR open until we have plan to release the new version of the Add-on?

**Comment by mahirchavda** [comment]
I think we can merge.

**Pull request merged**